### PR TITLE
use discord timestamps

### PR DESCRIPTION
**Pull request by Le0Developer**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ export function getDescription(incident: Incident) {
   for (const update of incident.incident_updates) {
     const time = new Date(update.created_at);
 
-    description += `\n**${pascalCase(update.status)}** - _${time.toUTCString()}_\n${update.body}\n`;
+    description += `\n**${pascalCase(update.status)}** - <t:${Math.floor(time / 1000)}:F>\n${update.body}\n`;
```
Comment (MattIPv4) [inline src/utils.ts:26]: Can we have both the relative timestamp and the UTC? For very old incis etc. the relative timestamp will become rather unhelpful

Comment (Le0Developer) [reply src/utils.ts:26]: This is not the relative timestamp, you are thinking of `:R>` at the end.
image

Comment (MattIPv4) [reply src/utils.ts:26]: Ah! My bad -- maybe relative + long would be useful for folks?

Comment (Le0Developer) [reply src/utils.ts:26]:
```suggestion
    description += `\n**${pascalCase(update.status)}** - <t:${Math.floor(time / 1000)}:F> (<t:${Math.floor(time / 1000)}:R>)\n${update.body}\n`;
```

File: src/utils.ts [hunk 2]
```diff
@@ -23,7 +23,7 @@ export function getDescription(incident: Incident) {
   for (const update of incident.incident_updates) {
     const time = new Date(update.created_at);
 
-    description += `\n**${pascalCase(update.status)}** - _${time.toUTCString()}_\n${update.body}\n`;
+    description += `\n**${pascalCase(update.status)}** - <t:${Math.floor(time / 1000)}:F> (<t:${Math.floor(time / 1000)}:R>)\n${update.body}\n`;
```
Comment (WalshyDev) [inline src/utils.ts:26]:
```suggestion
    const ms = Math.floor(time.getTime() / 1000);
    description += `\n**${pascalCase(update.status)}** - <t:${ms}:F> (<t:${ms}:R>)`
      + `\n${update.body}\n`;
```

**Review by WalshyDev — APPROVED** [review]

File: src/utils.ts (hunk 2 above)
Comment (Le0Developer) [reply src/utils.ts]: It's not milliseconds tho

**Pull request merged**